Content/src/Client/webpack.config.js

```diff
@@ -58,62 +87,79 @@ module.exports = {
+        ])
+        : commonPlugins.concat([
+            new webpack.HotModuleReplacementPlugin(),
+        ]),
+    resolve: {
+        // See https://github.com/fable-compiler/Fable/issues/1490
+        symlinks: false
+    },
+    // Configuration for webpack-dev-server
     devServer: {
-        // redirect requests that start with /api/* to the server on port 8085
-        proxy: {
-            '/api/*': {
-                target: 'http://localhost:' + (process.env.SERVER_PROXY_PORT || "8085"),
-                changeOrigin: true
-            }
-        },
-        // turn on hot module reloading
+        publicPath: "/",
+        contentBase: resolve(CONFIG.assetsDir),
+        port: CONFIG.devServerPort,
+        proxy: CONFIG.devServerProxy,
         hot: true,
-        // more automatic reloading
-        inline: true,
-        // default page
-        historyApiFallback: { index: resolve("./index.html") },
-        // where to server static files from
-        contentBase: resolve("./public")
+        inline: true
     },
-    // The modules that are used by webpack for transformations
+    // - fable-loader: transforms F# into JS
+    // - babel-loader: transforms JS to old syntax (compatible with old browsers)
+    // - sass-loaders: transforms SASS/SCSS into JS
+    // - file-loader: Moves files referenced in the code (fonts, images) into output folder
     module: {
         rules: [
             {
-                // - fable-loader: transforms F# into JS
                 test: /\.fs(x|proj)?$/,
                 use: {
                     loader: "fable-loader",
                     options: {
-                        babel: babelOptions,
-                        define: isProduction ? [] : ["DEBUG"]
-                   }
-                },
+                        babel: CONFIG.babel
+                    }
+                }
             },
             {
-                // - babel-loader: transforms JS to old syntax (compatible with old browsers)
                 test: /\.js$/,
                 exclude: /node_modules/,
                 use: {
                     loader: 'babel-loader',
-                    options: babelOptions
+                    options: CONFIG.babel
                 },
+            },
+            {
+                test: /\.(sass|scss|css)$/,
+                use: [
+                    isProduction
+                        ? MiniCssExtractPlugin.loader
+                        : 'style-loader',
+                    'css-loader',
+                    {
+                      loader: 'sass-loader',
+                      options: { implementation: require("sass") }
+                    }
+                ],
+            },
+            {
+                test: /\.(png|jpg|jpeg|gif|svg|woff|woff2|ttf|eot)(\?.*)?$/,
+                use: ["file-loader"]
             }
         ]
     }
 };
+
+function resolve(filePath) {
+    return path.isAbsolute(filePath) ? filePath : path.join(__dirname, filePath);
```

Comment:
You can simplify by using:

```fs
function resolve(filePath) {
    return path.join(__dirname, filePath)
}
```

Reply:
Same as above - the file comes from webpack-config-template